plugin/www/applovinmax.js

```diff
@@ -366,6 +366,9 @@ window.addEventListener('OnRewardedAdDisplayedEvent', function (adInfo) {
 window.addEventListener('OnRewardedAdFailedToDisplayEvent', function (adInfo) {
     AppLovinMAX.isAdReadyValues[adInfo.adUnitId] = false;
 });
+window.addEventListener('OnRewardedAdReceivedRewardEvent', function (adInfo) {
+    AppLovinMAX.isAdReadyValues[adInfo.adUnitId] = true;
```

Comment:
Should be false, since the ad is shown already so technically it is no longer ready.
```suggestion
    AppLovinMAX.isAdReadyValues[adInfo.adUnitId] = false;
```